ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -623,6 +623,7 @@ ibmcloud*)
     else
       IC_API_KEY="$(< "${CLUSTER_PROFILE_DIR}/ibmcloud-api-key")"
     fi
+    echo "setting IBMCloud Key: $IC_API_KEY"
```

Comment:
Do we want to but outputting the APIKey?

I see that the key itself gets obfuscated in the [output](https://qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/view/gs/qe-private-deck/pr-logs/pull/openshift_release/59392/rehearse-59392-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.18-installer-rehearse-ibmcloud-capi/1866434336299945984) at least.
>setting IBMCloud Key: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX

Reply:
I just confirmed the key is set, and it is the same as the vault (the value saved in  vault will be changed X)